assets/src/js/cookies.js

```diff
@@ -34,8 +34,6 @@ export const setupCookies = function($) {
   }
 
   $('#hidecookie').click(function () {
-    $('.cookie-notice').slideUp('slow');
```

Comment:
Removing this doesn't prevent the cookie bar from being hidden on button click?

Reply:
Yes, I don't know how I missed that. Replaced the slideUp with a simple fadeOut, not only because the slide was buggy, it also had a performance impact as it was pushing the entire layout.